### PR TITLE
Add Warning When Image Is Too Large

### PR DESCRIPTION
**Pull request by blessedcoolant**
Just a small warning to let the user know that their image is much bigger than default and that if they OOM, they need to make it smaller.

### CODE REVIEW

File: ldm/simplet2i.py [hunk 1]
```diff
@@ -158,8 +157,9 @@ def __init__(
         self.device = device
 
         # for VRAM usage statistics
-        self.session_peakmem = torch.cuda.max_memory_allocated() if self.device == 'cuda' else None
-            
+        self.session_peakmem = torch.cuda.max_memory_allocated(
+        ) if self.device == 'cuda' else None
+
```
Comment (tildebyte) [inline ldm/simplet2i.py:162]: Ugh. I have to strenuously disagree with 'blue' on this. It makes it *harder* to understand.

**Review by tildebyte — APPROVED** [review]
LGTM with comment

File: ldm/simplet2i.py (hunk 1 above)
Comment (lstein) [reply ldm/simplet2i.py:162]: Blue did that?

**Review by lstein — APPROVED** [review]
Looks good to me too.

**Review by lstein — APPROVED** [review]

**Comment by magnusviri** [comment]
Is "blue" a formatter?

**Comment by blessedcoolant** [comment]
> Is "blue" a formatter?

Yes

**Pull request merged**